package cloudutils4j.exceptions.s3.invalid.localpath;

public class NullDestinationLocalPathException extends RuntimeException {
    public NullDestinationLocalPathException(String message) { super(message); }
}
